### ISSUE TITLE
Paging information lost when viewing records from a Dashboard list

### ISSUE THREAD

**Issue by ShavaShav** [odoo/odoo#74126]
Impacted versions:
13 & 14 (possibly earlier as well but I haven't tested)

Steps to reproduce:

1. Go to any app and open a tree view (such as Sale Orders list) with multiple records
2. Click "Favorites" in the control panel, and then "Add to my Dashboard"
3. Go to Dashboard app
4. Click one of the records in the list which should open the form view

Current behavior:
The pager in the form view shows "1/1", and user isn't able to page through the records like they would elsewhere.

Expected behavior:
The pager in the form view shows "1/total", where total is the length of the dashboard's list view. Intuitively, the pager should function the same as if we are viewing the record from the original list view.

Video/Screenshot link (optional):
List view:
![image](https://user-images.githubusercontent.com/17697598/126656738-a4042f15-e899-4c64-8f70-eb48c79a4cd7.png)

Clicking on a record opens the form view with correct paging:
![image](https://user-images.githubusercontent.com/17697598/126656848-ff4e69f3-9a5d-4fbd-8bd6-0f462b25ba81.png)

Resulting dashboard view:
![image](https://user-images.githubusercontent.com/17697598/126656781-70111303-3f88-47c0-99e6-237da5157d9a.png)

Clicking on a dashboard record opens the form view without paging. It should say "1/3" instead like above:
![image](https://user-images.githubusercontent.com/17697598/126656893-84f565c5-3b91-42c7-8d0f-bb46f8e3b279.png)

Support ticket number submitted via odoo.com/help (optional):
N/A

I did some digging in the code and found that the Dashboard's [onSwitchView starts a new action](https://github.com/odoo/odoo/blob/13.0/addons/board/static/src/js/board_view.js#L102) to purposefully clear the breadcrumbs and I suspect this is also the reason for losing pagination, but I can't figure out how to restore it.

**Comment by pedrobaeza** [comment]
You are right about the issue description, but not sure if it can be solved. I tag for the suitable Odoo team.

**Comment by TTPBruce** [comment]
This issue still exists in Odoo 15.

**Comment by gerardvilajuansmc** [comment]
Hello, I just saw this open issue, and it seems to be the same. We are using Odoo 16, and we noticed that in a custom view (model + view), if there are multiple items inside, pagination tracking stops working.
<img width="145" alt="Image" src="https://github.com/user-attachments/assets/fb911628-bfe5-4969-8d36-13f26810e5be" />
<img width="101" alt="Image" src="https://github.com/user-attachments/assets/7b010ebc-91e1-4842-9e9c-fe1d1a1a89ca" />

When I enter an object in the view and click on an item, the tracking stops, and I can no longer navigate through the records.

This is likely related to the same issue, but I didn't found more information apart of this post.
